### ISSUE TITLE
Error Build Version

### ISSUE THREAD

**Issue by fbsanches** [fbsanches/cordova-plugin-wallpaper#17]
I do not know what I did. I do not have many experiences with git, travis and others. It looks like it has errors. How to correct?

**Comment by janbaudisch** [comment]
I could try to solve this with TravisCI, but maybe the best thing would be to switch CIs. I think you should switch to [CircleCI](https://circleci.com/). In that way we could use a docker image for builds. It is just that I could handle CircleCI much better. I would provide a configuration for that and so on. Just let me know if you want to stick with Travis or if you would switch to CirlceCI.

**Comment by fbsanches** [comment]
Whatever you think is best, my friend. Feel free. Together we can make this plugin a lot better. I had the idea and I started, but now I need people with more knowledge for this and you have been helping a lot. Feel free. What do you think is best?

**Comment by janbaudisch** [comment]
Well, CircleCI is better. So you could sign up at CircleCI with your GitHub account and start building the project. I will do the other stuff.

**Comment by fbsanches** [comment]
Ready... 

https://circleci.com/gh/fbsanches/cordova-plugin-wallpaper

Anything else or is it enough already?

**Comment by janbaudisch** [comment]
This is enough. I will test my docker image and write a configuration file and we should be done.

**Comment by janbaudisch** [comment]
I still need some time with the docker image. When it is ready, I will make the changes to this project.

**Comment by fbsanches** [comment]
Error Build https://circleci.com/gh/fbsanches/cordova-plugin-wallpaper/5

**Comment by janbaudisch** [comment]
Yes, have also seen that. Something is stoll wrong with the configuration. I will create a test reposirtoy and test the configuration there. Once it's working I am going to make the changes here.

**Comment by fbsanches** [comment]
Great job.

**Comment by janbaudisch** [comment]
Yay, tests are passing! Alright, this should be fixed.
